### PR TITLE
Add font weight global variables

### PR DESCRIPTION
**Pull request by jlukic**
Adds font weight variables ala #6167

### CODE REVIEW

**Comment by jaridmargolin** [comment]
Heyo!!!! Thank you for getting this in so fast 🙌

Just to ensure I understand how this differs from the original PR.

1. `@fontWeightNormal` -> `@normal`
2. `@fontWeightBold` -> `@bold`
3. Remove `@fontWeightIcon` (wasn't entirely sure if this was necessary in the first place)
4. Only update `bold/normal` values within the `default` theme (I had made updates across all themes)

Thanks again :)

**Comment by jlukic** [comment]
You can now just put in a component's theme variables `@bold: 700;` to modify how it handles bold.

To modify it globally add to `site.variables` for site or theme.

File: src/themes/default/collections/table.variables [hunk 1]
```diff
@@ -61,8 +61,8 @@
 @footerColor: @textColor;
 @footerVerticalPadding: @cellVerticalPadding;
 @footerHorizontalPadding: @cellHorizontalPadding;
-@footerFontStyle: normal;
-@footerFontWeight: normal;
+@footerFontStyle: @normal;
```
Comment (ktmn) [inline src/themes/default/collections/table.variables]: `@footerFontStyle: @normal;`

Font style using font weight value.

File: src/themes/default/elements/button.variables [hunk 2]
```diff
@@ -152,8 +152,8 @@
 
 @orBackgroundColor: @white;
 @orTextShadow: @invertedTextShadow;
-@orTextStyle: normal;
-@orTextWeight: bold;
+@orTextStyle: @normal;
```
Comment (ktmn) [inline src/themes/default/elements/button.variables]: `@orTextStyle: @normal;`

Text style using font weight value.

File: src/themes/default/modules/popup.variables [hunk 3]
```diff
@@ -17,8 +17,8 @@
 
 @verticalPadding: 0.833em;
 @horizontalPadding: 1em;
-@fontWeight: normal;
-@fontStyle: normal;
+@fontWeight: @normal;
+@fontStyle: @normal;
```
Comment (ktmn) [inline src/themes/default/modules/popup.variables]: `@fontStyle: @normal;`

Font style using font weight value.

File: src/themes/default/views/feed.variables [hunk 4]
```diff
@@ -54,13 +54,13 @@
 @datePadding: 0em;
 @dateColor: @lightTextColor;
 @dateFontSize: @relativeMedium;
-@dateFontWeight: normal;
-@dateFontStyle: normal;
+@dateFontWeight: @normal;
+@dateFontStyle: @normal;
```
Comment (ktmn) [inline src/themes/default/views/feed.variables]: `@dateFontStyle: @normal;`

Font style using font weight value.

**Pull request merged**